vllm/model_executor/layers/quantization/compressed_tensors/schemes/compressed_tensors_w4a16.py

```diff
@@ -0,0 +1,159 @@
+from typing import Callable, List, Optional
+
+import torch
+from torch.nn import Parameter
+
+from vllm._C import ops
+from vllm.model_executor.layers.quantization.compressed_tensors.schemes import (
+    CompressedTensorsScheme)
+from vllm.model_executor.layers.quantization.gptq_marlin import (
+    GPTQMarlinState, marlin_permute_scales)
+from vllm.model_executor.utils import set_weight_attrs
+
+__all__ = ["CompressedTensorsW4A16"]
+
+
+class CompressedTensorsW4A16(CompressedTensorsScheme):
```

Comment:
Do you think this should be W4A16 or WNA16?

- i.e. will the W8A16 kernels be hooked in here or do you expect to make another `Scheme`?